src/utils/get-promo-type/index.js

```diff
@@ -0,0 +1,26 @@
+function getPromoType(content) {
+	if (!content) {
+		return undefined;
+	}
+
+	let promoType = "";
+
+	switch (content.type) {
```

Comment:
@matthewroach : Do we not need to include "image" as a case here?

Reply:
I've added `image` as a use case - as this function is a port of `discoverPromo` from blocks I didn't expand the scope of it for this PR in the first instance, but adding image seems like it will not hurt